examples/exex/op-bridge/src/main.rs

```diff
@@ -16,18 +11,23 @@ use rusqlite::Connection;
 sol!(L1StandardBridge, "l1_standard_bridge_abi.json");
 use crate::L1StandardBridge::{ETHBridgeFinalized, ETHBridgeInitiated, L1StandardBridgeEvents};
 
-/// An example of ExEx that listens to ETH bridging events from OP Stack chains
-/// and stores deposits and withdrawals in a SQLite database.
-struct OPBridgeExEx<Node: FullNodeTypes> {
+/// Initializes the ExEx.
+///
+/// Opens up a SQLite database and creates the tables (if they don't exist).
+async fn init<Node: FullNodeTypes>(
     ctx: ExExContext<Node>,
-    connection: Connection,
+) -> eyre::Result<impl Future<Output = eyre::Result<()>>> {
+    let mut connection = Connection::open("op_bridge.db")?;
```

Comment:
let's pass it as an argument, I want to add a test with passing the in-memory connection later